controllers/managedocs_controller.go

```diff
@@ -895,3 +903,116 @@ func (r *ManagedOCSReconciler) own(resource metav1.Object) error {
 	}
 	return nil
 }
+
+func (r *ManagedOCSReconciler) reconcileRookCephOperatorConfig() error {
+	rookConfigMap := &corev1.ConfigMap{}
+	rookConfigMap.Name = rookConfigMapName
+	rookConfigMap.Namespace = r.namespace
+
+	if err := r.get(rookConfigMap); err != nil {
+		// Because resource limits will not be set, failure to get the Rook ConfigMap results in failure to reconcile.
+		return fmt.Errorf("Failed to get Rook ConfigMap: %v", err)
+	}
+
+	if rookConfigMap.Data == nil {
+		rookConfigMap.Data = make(map[string]string)
+	}
+
+	rbdProvisionerLimits := utils.MarshalRookResourceRequirements([]utils.RookResourceRequirements{
```

Comment:
The `RookResourceRequirements` is used as a list in more than one place. 
I will suggest to define the following in the util file and use it instead
```
type RookResourceRequirementsList []RookResourceRequirements
```